src/software/SfM/main_ComputeStructureFromKnownPoses.cpp

```diff
@@ -117,11 +111,23 @@ int main(int argc, char **argv)
 
   openMVG::system::Timer timer;
 
+  // Clear previous 3D landmarks
+  sfm_data.structure.clear();
+
   //------------------------------------------
   // Compute Structure from known camera poses
   //------------------------------------------
   SfM_Data_Structure_Estimation_From_Known_Poses structure_estimator;
-  structure_estimator.run(sfm_data, pairs, regions_provider);
+  structure_estimator.match(sfm_data, pairs, regions_provider);
+
+  // Unload descriptors before triangulation
+  regions_provider->clearDescriptors();
```

Comment:
is it just to save memory as the descriptors are not used anymore from that point on?
Maybe it can be moved directly into `run()`?